soccer/SystemState.hpp

```diff
@@ -70,9 +70,14 @@ class SystemState {
      */
 
     /** @ingroup drawing_functions */
-    void drawLine(const Geometry2d::Line& line, const QColor& color = Qt::black,
+    void drawLine(const Geometry2d::Segment& line,
```

Comment:
Meh, I guess it's ok as is.  I don't particularly like it, but it's not really causing any issues.  Maybe just put a comment above drawLine and drawSegment to clarify what each does.